test/csharp/edge/test.ts

```diff
@@ -28,6 +28,12 @@ describe("csharp full integration test synth", () => {
       stdio: "inherit",
       env: driver.env,
     });
+
+    execSync("dotnet add package Constructs", {
+      stdio: "inherit",
+      env: driver.env,
+    });
+
```

Comment:
@DanielMSchmidt feel free to pull this out into a PR so constructs updates stop breaking tests.

Reply:
Done, thank you! https://github.com/hashicorp/terraform-cdk/pull/1460